fakestorage/json_response.go

```diff
@@ -53,3 +55,12 @@ func (r *jsonResponse) getErrorMessage(status int) string {
 	}
 	return http.StatusText(status)
 }
+
+func errToJsonResponse(err error) jsonResponse {
+	// TODO: map more errno
```

Comment:
I think we can remove this TODO and just map errors on demand. What do you think?

Reply:
Done.